README.md

```diff
@@ -48,4 +48,12 @@ fluri
   ..query = 'foo=true';
 
 fluri.updateQuery({'bar': '10'});
-```
\ No newline at end of file
+```
+
+
+## Development
+
+Truss leverages [the dart_dev package](https://github.com/Workiva/dart_dev) for
```

Comment:
For copy/paste friendliness, could just get in the habit of saying `This project leverages...`